### DISCUSSION TITLE
Search Filter

### DISCUSSION THREAD

**Discussion by warengonzaga** [web3phl/directory#40] [Brainstorm]
Just wanna add here this for open discussion about the possible approach for the search filter. I'll lean this to @jnebab as he leads the front-end development of the directory. 🤝

**Comment by warengonzaga** [comment]
**UPDATE**: currently posted as an issue here #39

**Comment by jnebab** [comment]
hey guys, should we include filter by tags with this issue? If so, how would the user flow would look like? I am thinking if we can display all possible tags below the search box and when a user taps/click one tag, it becomes highlighted and call the filter API

**Reply by warengonzaga** [reply]
Good question, I think we should have a standard tags that can cater all projects. cc: @kquirapas

**Comment by Marveeeen** [comment]
Hi, is this implemented? If no and there's a design guide for this, I am willing to help

**Reply by warengonzaga** [reply]
hey @jnebab maybe you are able to help here? Unfortunately, we might create a new web app for this and include the directory inside. cc: @kquirapas

**Reply by jnebab** [reply]
if we plan to create a new web app, then it's better to not continue this feature and just add it as a new addition of the new web app :)

**Reply by warengonzaga** [reply]
We decided not to continue this @jnebab we're working on a new project and we'll move it to a new platform.
